stable/goldilocks/values.yaml

```diff
@@ -174,3 +174,7 @@ dashboard:
   affinity: {}
   # dashboard.topologySpreadConstraints -- Topology spread constraints for the dashboard pods
   topologySpreadConstraints: []
+
+deployment:
+  additionalLabels:
+  additionalPodLabels:
```

Comment:
I think this was supposed to be in stable/fairwinds-insights/values.yaml 😄

Reply:
done